db.js

```diff
@@ -13,11 +13,11 @@ const port = process.env.MICRO_DB_PORT || process.env.PORT || 9001;
 var dataStore = {}
 var oldestNotFinishedTask = 0
 
-if (process.argv.length < 3) {
-    console.log("Usage: node db.js <KEYVALUESTORE_ENDOPOINT>")
+kvEndpoint = common.getKeyvaluestoreEndpoint();
```

Comment:
Since `common` is only used here, I wonder if it would be "cleaner" if you didn't have the `require` above and would simply use `kvEndpoint = require('common').getKeyvaluestoreEndpoint();`